### PR TITLE
In the Margins Sidebar Presence Fix

### PR DESCRIPTION
**Pull request by danny98m**
## Sidebar's Presence is Now Made More Apparent to the User
This feature is a fix to issue [#696](https://github.com/WheatonCS/Lexos/issues/696)
When the user comes to the upload page, the sidebar will pop out for a little bit and then return to its original position to make its presence more known. This only happens on the upload page as I figured it would be annoying if it happened on every page but if that is better it's a simple change. 

Here is a gif with terrible quality detailing the animation (The animation would only happen once I am just continuously reloading the upload page so it happens a lot):

### CODE REVIEW

**Comment by scottkleinman** [comment]
That looks pretty close to a Bootstrap popover, albeit with prettier styling. Another approach would be to use an alert. But the main thing is to get the trigger code right (only when the session is created). We can then get creative with the appearance.

**Comment by chantisnake** [comment]
Typical website use a persistent cookies to see if a popup has been displayed. We can try and do that?

**Comment by scottkleinman** [comment]
Another possibility is [web storage](https://www.w3schools.com/html/html5_webstorage.asp).

File: lexos/static/js/scripts_upload.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
+$(document).ready(function () {
```
[the remark this answers is not in the document]
Comment (danny98m) [reply lexos/static/js/scripts_upload.js]: done.

**Comment by Weiqi97** [comment]
Close this pull request since this solution does not work on chrome.

**Pull request closed without merging**